src/Components/Endpoints/src/Rendering/EndpointHtmlRenderer.Streaming.cs

```diff
@@ -190,7 +191,7 @@ protected override void RenderChildComponent(TextWriter output, ref RenderTreeFr
     {
         var componentId = componentFrame.ComponentId;
         var sequenceAndKey = new SequenceAndKey(componentFrame.Sequence, componentFrame.ComponentKey);
-        WriteComponentHtml(componentId, output, allowBoundaryMarkers: true, sequenceAndKey);
+        WriteComponentHtml(componentId, output, _allowBoundaryMarkers, sequenceAndKey);
```

Comment:
Have you checked this definitely works? I'm suspicious this might have no effect at all, since `allowBoundaryMarkers` controls insertion of markers for streaming SSR updates, not markers for interactive components.

It looks like the E2E test doesn't say anything about activating interactive components so it's hard to be sure what will happen :)

Reply:
Good catch, I've updated this to be more thorough.

We now omit any markers and disable interactive components